bin/hdfcoinc/pycbc_make_coinc_search_workflow

```diff
@@ -387,6 +404,16 @@ if len(files_for_combined_injfind) > 0:
     inj = wf.make_foundmissed_plot(workflow, found_inj, 
                 rdir['injections'], require='summ', tags=['ALLINJ'])
     inj_summ = list(layout.grouper(inj + sen, 2))
+
+# make segment summary and copy segment files into results pages
+analysis_time_summ = ([(time_file,)] + [(seg_summ_plot,)] + 
+                      [(seg_summ_table, veto_summ_table)])
+layout.two_column_layout(rdir['analysis_time'], analysis_time_summ)
+wf.makedir(rdir['analysis_time/segment_data'])
+for seg_file in os.listdir("segments/"):
+    if seg_file.endswith(".xml") or seg_file.endswith(".xml.gz"):
+      shutil.copy(os.path.join("segments",seg_file),
+                  os.path.join(rdir['analysis_time/segment_data'],seg_file))
                             
```

Comment:
Don't use copy here. Instead use a symlink as we do for the main summary section. That is valid before the files exists, and won't require us to keep the file in two places.